packages/core/admin/admin/src/content-manager/components/BlocksEditor/Toolbar/index.js

```diff
@@ -120,6 +78,9 @@ const isBlockActive = (editor, matchNode) => {
 };
 
 const toggleBlock = (editor, value) => {
```

Comment:
this whole function is really unclear what it's supposed todo, it looks like it's just fighting against slate – can we keep in mind other engineers outside the current squad will maintain this and write descriptive names & comments to help people please?

Reply:
This will be handled on Simone's PR https://github.com/strapi/strapi/pull/18189/files